### PR TITLE
Add Doctor Fortran blog to learning.yml

### PR DESCRIPTION
**Pull request by sblionel**
Add Steve Lionel's "Doctor Fortran" blog to the list of external resources on the Fortran language

### CODE REVIEW

**Comment by milancurcic** [comment]
#build_preview

File: _data/learning.yml [hunk 1]
```diff
@@ -67,8 +67,10 @@ reference-links:
   - name: "Scivision Fortran 2018 Examples"
     url: https://github.com/scivision/fortran2018-examples
     description: A github repository containing code samples for various Fortran 2018 features
-
-
+    
+  - name: "Doctor Fortran Blog"
+    url: https:/stevelionel.com/drfortran
```
Comment (LKedward) [inline _data/learning.yml:72]:
```suggestion
    url: https://stevelionel.com/drfortran
```

Comment (sblionel) [reply _data/learning.yml:72]: Oh crud - thanks for noticing.

**Comment by certik** [comment]
Thanks!

**Comment by milancurcic** [comment]
#delete_preview

**Pull request merged**